### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by maguireong**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -266,7 +266,17 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `FitEgo` and the **Actor** is the `user`, unless specified otherwise)
+
+
+**Use case: Add a Client**
+
+**MSS**
+
+1.  User requests to add a specific Client in the list
+2.  FitEgo add the Client
```
Comment (benclmnt) [inline docs/DeveloperGuide.md]:
```suggestion
2.  FitEgo adds the Client
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -357,6 +366,14 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Adding a Client
+
+1. Adding a Client while all Clients are being shown
+
+   1. Test case: `cadd n/David …​` <br>
+      Expected: First contact is added to the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
```
Comment (benclmnt) [inline docs/DeveloperGuide.md]: I'm not too sure if there is a timestamp in the status bar 😅 , but it is also in the AB3 DG ...

Comment (benclmnt) [inline docs/DeveloperGuide.md]:
```suggestion
      Expected: First contact is added to the list. Details of the added contact shown in the status message. Timestamp in the status bar is updated.
```

**Review by benclmnt — CHANGES_REQUESTED** [review]
Hi, there is some copy-paste error... but other than that its good to go

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (maguireong) [reply docs/DeveloperGuide.md]: hmm but for now, we are still largely following AB3 so it should still be there?

**Review by benclmnt — APPROVED** [review]
lgtm

**Pull request merged**